infrastructure/private-api-config.yaml

```diff
@@ -0,0 +1,80 @@
+plugin: openapi
+specFile: ./private-api.yaml
+
+validation:
+  request: log
+
+resources:
+  - method: GET
+    path: /authorization
+    requestHeaders:
+      SessionHeader: session-id
+    response:
+      statusCode: 200
+
+  - method: GET
+    path: /authorization
+    requestHeaders:
+      SessionHeader: ""
+    response:
+      statusCode: 400
+
+  - method: GET
+    path: /authorization
+    requestHeaders:
+    response:
+      statusCode: 500
+
+  - method: POST
+    path: /session
+    requestBody:
+      jsonPath: $.client_id
+      value: ipv-core-stub
+    requestHeaders:
+      X-Forwarded-For: test
+    response:
+      statusCode: 201
+
+  - method: POST
+    path: /session
+    requestBody:
+      value: ""
+    response:
+      statusCode: 400
+
+  - method: POST
+    path: /session
+    requestBody:
+      value: invalid
+    response:
+      statusCode: 500
+
+  - method: POST
+    path: /favourite
+    requestHeaders:
+      SessionHeader: session-id
+    requestBody:
+      jsonPath: $.toy
+      value: football
```

Comment:
Can these examples match the third party stub spec we've got for the toy api - https://github.com/alphagov/di-ipv-third-party-stubs/blob/main/stubs/toy/api-config.yaml